OJP/OJP_Trips.xsd

```diff
@@ -106,18 +107,54 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
-	<xs:group name="TripPolicyGroup">
+	<xs:complexType name="AccessModesListOfEnumerations">
 		<xs:annotation>
-			<xs:documentation>Policies that control the trip search behaviour.</xs:documentation>
+			<xs:documentation>List of Access Modes.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:group ref="BaseTripPolicyGroup"/>
-			<xs:element name="ItModesToCover" type="IndividualModesEnumeration" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="AccessMode" type="ifopt:AccessModesEnumeration" minOccurs="0" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
+					<xs:documentation>Access mode that should be considered.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
+	</xs:complexType>
+	<xs:group name="TripPolicyGroup">
+		<xs:annotation>
+			<xs:documentation>Policies that control the trip search behaviour.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:group ref="BaseTripPolicyGroup"/>
+			<xs:choice>
+				<xs:element name="MotorizedMainTravelMode" type="xs:boolean" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>The MotorizedMainTravelMode defines whether the trip planner works in PT mode (or pure IT mode) or includes carTransportRail and the like. If set to true (e.g. because one travels by car, truck or motorcycle), then carTransportRail, ferry are used together with roads.</xs:documentation>
```

Comment:
```suggestion
						<xs:documentation>The MotorisedMainTravelMode defines whether the trip planner works in PT mode (or pure IT mode) or includes carTransportRail and the like. If set to true (e.g. because one travels by car, truck or motorcycle), then carTransportRail, ferry are used together with roads.</xs:documentation>
```